ktor-client/ktor-client-core/common/src/io/ktor/client/request/builders.kt

```diff
@@ -110,18 +137,64 @@ public suspend inline fun <reified T> HttpClient.head(builder: HttpRequestBuilde
     return request(builder)
 }
 
+/**
+ * Executes a [HttpClient] GET request, with the information from the [builder]
+ */
+public suspend fun HttpClient.getRaw(builder: HttpRequestBuilder): HttpResponse = get(builder)
```

Comment:
We still should add these names to the design agenda. It is simpler but still not the simplest naming

Reply:
updated design ticket